frontend/components/events/ShareModal.tsx

```
@@ -108,25 +107,33 @@ ${eventURL}`;
-                      <li className="cursor-pointer" onClick={handleEmailShare} title="Share via Email">
-                        <Image src={email_icons} alt="Share via Email" className="h-11 w-11" />
-                      </li>
-                    </ul>
-                    <p id="url" onClick={highlightURL} className="mb-4 cursor-pointer">
-                      {eventURL}
-                    </p>
-                    <button className="bg-blue-200 px-2 py-1 rounded" onClick={copyURL}>
-                      Copy
-                    </button>
+                  <div className="mt-2 flex items-center justify-center gap-4">
+                    <div className="cursor-pointer" onClick={handleFacebookShare} title="Share on Facebook">
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)